src/ecc.rs

```diff
@@ -265,7 +268,7 @@ impl Ecc {
             buf.put_u8(self.transport.put_command_flag());
             command.bytes_into(&mut buf);
 
-            self.transport.send_wake(wake_delay)?;
+            self.transport.send_wake(wake_delay, Duration::from_millis(self.config.min_wake_interval.into()))?;
```

Comment:
* Move Duration::from to out of loop (below wake_delay)
* Use from_micros for consistency with the other timing values